test/integration/discovery.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+load helpers
+
+DISCOVERY_FILE=/tmp/test_file_discovery
```

Comment:
Could we use `mktemp` instead?

The goal of the integration tests is to run them on Jenkins or something like that automatically after each merge, so we kinda control the environment.